lib/cylc/task_events_mgr.py

```diff
@@ -553,7 +552,7 @@ def _process_job_logs_retrieval(self, schd_ctx, ctx, id_keys):
             "retrieve job logs command", s_host, s_user))
 
         cmd = shlex.split(rsync_str) + ["--rsh=" + ssh_str]
-        if cylc.flags.debug:
+        if LOG.level <= DEBUG:
```

Comment:
Maybe (not tested) `LOG.isEnabledFor(logging.DEBUG))` ? It does one extra check I think?

```python
# from my current logging/__init__.py in Py2.7, class Logger
    def isEnabledFor(self, level):
        """
        Is this logger enabled for level 'level'?
        """
        if self.manager.disable >= level:
            return 0
        return level >= self.getEffectiveLevel()
```

Reply:
Done.